todo_app/data/trello_items.py

```diff
@@ -0,0 +1,95 @@
+import requests,os, json
+from dotenv import load_dotenv, find_dotenv
+
+load_dotenv(find_dotenv())
+
+key = os.getenv('TRELLO_API_KEY') 
```

Comment:
Similarly to above, these variables are set when this file is _imported_, which works nicely here, but might catch you out in M3! When it comes to that module, the simplest solution is probably to shift these into the functions, e.g. duplicate them in each relevant function:
```
def get_board_id():
    fields = 'fields=name'
    key = os.getenv('TRELLO_API_KEY')
    token = os.getenv('TRELLO_API_TOKEN')
    url = 'https://api.trello.com/1/members/me/boards?{}&key={}&token={}'.format(fields,key,token)
```
This delays them, so that we only load from the environment when we are ready and the functions are being called, rather than when this file is first imported